htdocs/map.js

```diff
@@ -340,6 +344,15 @@ $(function(){
         return infowindow;
     }
 
+    var linkifyCallsign = function(callsign) {
+        if ((callsign_url == null) || (callsign_url == ''))
+            return callsign;
+        else
+            return '<a href="#" onclick="window.open(' + "'" +
+                callsign_url.replaceAll('{}', callsign) +
+                "','callsign_info'" + ');">' + callsign + '</a>';
+    };
```

Comment:
Shouldn't this be possible without javascript? A simple `<a href="..." target="_blank">` normally does the trick.

Reply:
The change you are suggesting will open a new window every time. My original code will reuse the same window, once open.

Reply:
If you want to reuse the same tab, change `target` to a frame name (that is not one of the reserved targets).

Reply:
This apparently does not work on some browsers, but I have pushed the change you requested.

Reply:
It would surprise me, but if that's the case then it probably will not matter in this particular instance: The link that leads to the map is done the same way.